test/module/irohad/ametsuchi/wsv_query_command_test.cpp

```diff
@@ -126,11 +131,12 @@ namespace iroha {
      */
     TEST_F(RolePermissionsTest, InsertRolePermissionsWhenNoRole) {
       auto new_role = role + " ";
-      ASSERT_TRUE(err(command->insertRolePermissions(new_role, {permission})));
+      ASSERT_TRUE(
+          err(command->insertRolePermissions(new_role, role_permissions)));
 
       auto permissions = query->getRolePermissions(new_role);
       ASSERT_TRUE(permissions);
-      ASSERT_EQ(0, permissions->size());
+      ASSERT_NE(*permissions &= role_permissions, role_permissions);
```

Comment:
`*permissions &= role_permissions` looks quite cryptic, any way to make it more readable?